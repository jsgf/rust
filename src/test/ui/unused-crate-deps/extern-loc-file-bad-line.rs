// --extern-location with a file/line reference

// check-pass
// aux-crate:bar=bar.rs
// compile-flags:--extern-location bar=file:{{src-base}}/unused-crate-deps/test.mk:300

#![warn(unused_crate_dependencies)]
//~^ WARNING external crate `bar` unused in

fn main() {}
