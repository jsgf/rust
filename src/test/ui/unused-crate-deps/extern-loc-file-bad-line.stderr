warning: external crate `bar` unused in `extern_loc_file_bad_line`: remove the dependency or add `use bar as _;`
  --> $DIR/extern-loc-file-bad-line.rs:7:1
   |
LL | #![warn(unused_crate_dependencies)]
   | ^
   |
note: the lint level is defined here
  --> $DIR/extern-loc-file-bad-line.rs:7:9
   |
LL | #![warn(unused_crate_dependencies)]
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^
   = help: remove unnecessary dependency `bar` at `$DIR/test.mk:300`

